samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -172,7 +204,7 @@ void download(struct addrinfo *ai)
 	}
 
 error:
-	(void)close(sock);
+	close(sock);
```

Comment:
Why this change?

Reply:
Must have slip in accidently, will revert that.